wp-content/plugins/star-wars/index.php

```diff
@@ -1,3 +1,130 @@
 <?php
 
-//STAR WARS PLUGIN GOES HERE
\ No newline at end of file
+/**
+ * @package Start-Wars
+ * @version 1.0
+ */
+/*
+Plugin Name: Star Wars
+Description: This plug-in is a challenge to apply for Developer Job in StickerStock. The plugin allows to randomly show in the any page top a Star Wars character from a list getting from Star Wars API https://swapi.dev/. 
+Author: Marco Portero
+Version: 1.0
+Author URI: https://www.linkedin.com/in/marcoportero/
+*/
+
+function star_wars_get_people() {
+
+	$people = 'Nobody';
+    // Get all characters from Star Wars API
+	$url = 'https://swapi.dev/api/people';
+
+      $curl = curl_init();
+      curl_setopt_array($curl, array(
+        CURLOPT_URL => $url,
+        CURLOPT_RETURNTRANSFER => true,
+        CURLOPT_FOLLOWLOCATION => true,
+        CURLOPT_ENCODING => "",
+        CURLOPT_MAXREDIRS => 10,
+        CURLOPT_SSL_VERIFYPEER => false,
+        CURLOPT_SSL_VERIFYHOST => false,  
+        CURLOPT_TIMEOUT => 30,
+        CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
+        CURLOPT_CUSTOMREQUEST => "GET"
+      ));
+      $response = curl_exec($curl);
+      $err = curl_error($curl);
+      curl_close($curl);
+      if ($err) {
+        //Only show errors while testing
+        printf( "cURL Error #: %s", $err);
+
+      } else {
+        
+        $responseObj = json_decode($response);
+        $people = $responseObj->results;
+        
+      }
+    
+    // Get a random position from the array
+    $pos = mt_rand( 0, count( $people ) - 1 );
```

Comment:
`$pos = mt_rand( 0, count( $people ) - 1 );` will only display characters from 1 to 10 in random order. 

Hint: `$people = $responseObj->results;` might have the key to properly display a truly random order of all characters from the API endpoint.